ci-scripts/schedule_k8s_deploy.sh

```diff
@@ -9,34 +9,46 @@ branch=${TEST_BRANCH-main}
   -e "AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY" \
   -v ~/.aws:/root/.aws \
   -v "$PWD":/aws \
   amazon/aws-cli dynamodb put-item --table-name "ceramic-$network-ops" --item \
-  "{                                                        \
-    \"id\":     {\"S\": \"$id\"},                           \
-    \"job\":    {\"S\": \"$job_id\"},                       \
-    \"ts\":     {\"N\": \"$now\"},                          \
-    \"ttl\":    {\"N\": \"$ttl\"},                          \
-    \"stage\":  {\"S\": \"queued\"},                        \
-    \"type\":   {\"S\": \"workflow\"},                      \
-    \"params\": {                                           \
-      \"M\": {                                              \
-        \"name\":     {\"S\": \"Deploy CERAMIC ONE\"},      \
-        \"org\":      {\"S\": \"3box\"},                    \
-        \"repo\":     {\"S\": \"ceramic-infra\"},           \
-        \"ref\":      {\"S\": \"$branch\"},                 \
-        \"workflow\": {\"S\": \"update_image.yml\"},        \
-        \"labels\":   {\"L\": [{\"S\": \"deploy\"}]},       \
-        \"inputs\":   {                                     \
-          \"M\": {                                          \
-            \"ceramic_one_image\": {\"S\": \"$image\"},     \
-            \"environment\": {\"S\": \"$environment\"}      \
-          }                                                 \
-        }                                                   \
-      }                                                     \
-    }                                                       \
+  "{                                                            \
+    \"id\":     {\"S\": \"$id\"},                               \
+    \"job\":    {\"S\": \"$job_id\"},                           \
+    \"ts\":     {\"N\": \"$now\"},                              \
+    \"ttl\":    {\"N\": \"$ttl\"},                              \
+    \"stage\":  {\"S\": \"queued\"},                            \
+    \"type\":   {\"S\": \"workflow\"},                          \
+    \"params\": {                                               \
+      \"M\": {                                                  \
+        \"name\":     {\"S\": \"Deploy CERAMIC ONE\"},          \
+        \"org\":      {\"S\": \"3box\"},                        \
+        \"repo\":     {\"S\": \"ceramic-infra\"},               \
+        \"ref\":      {\"S\": \"$branch\"},                     \
+        \"workflow\": {\"S\": \"update_image.yml\"},            \
+        \"labels\":   {\"L\": [{\"S\": \"deploy\"}]},           \
+        \"inputs\":   {                                         \
+          \"M\": {                                              \
+            \"ceramic_one_image\": {\"S\": \"$image\"},         \
+            \"ceramic_image\":     {\"S\": \"$ceramic_image\"}, \
+            \"environment\": {\"S\": \"$environment\"}          \
```

Comment:
minor: can you align the the `{` for `environment` with the braces above it?

Reply:
updated!